spec/std/json/serialization_spec.cr

```diff
@@ -127,19 +127,19 @@ describe "JSON serialization" do
       tuple.should be_a(NamedTuple(x: Int32?, y: String))
     end
 
-    pending_win32 "does for BigInt" do
+    it "does for BigInt" do
```

Comment:
AFAIK libgmp is still not available on windows ci. Why this change then?

Reply:
Sorry, I'm a newcomer, not aware of the windows setup, just changed it back.